### ISSUE TITLE
[BUG REPORT] Gremlin 2 - Shinshu Tanjou // Garbled merchand texts (SUNSOFT-FME-7 - Mapper 69)

### ISSUE THREAD

**Issue by Zet-sensei** [SourMesen/Mesen#633]
After a rapid test with Gremlin 2 - Shinshu Tanjou (Japan) - **CRC32 : ea313dc5**, I've seen a graphical error when entering the merchand's shop on all levels of the game.

_BUG -_ The Japanese text doesn't render correctly (garbled graphics).
_EXPECTED BEHAVIOUR_ - Correct render of the Japanese texts.

_HOW TO REPRODUCE -_ Use the following attached save state with the same ROM & enter through the door.
http://dl.free.fr/g5nQgVfBV

**Tested & confirmed on 0.9.7 and 0.9.7.119 versions.**

**NOTES -** The European & USA ROM versions aren't affected (NES-TLROM - Mapper 4).
So the bug is located only on mapper 69.

![Gremlin II Bug Report](https://user-images.githubusercontent.com/52127625/59975877-a28cba00-95bd-11e9-9350-16d0c54b5a19.png)

**Comment by retro-wertz** [comment]
this CRC ea313dc5 is not found in mesen's database. you probably have a rom with invalid header. looks like mirroring issue.

**Comment by SourMesen** [comment]
The game appears to work properly on my end:

```
File CRC32: 0xEA313DC5
PRG CRC32: 0x11FC8686
PRG+CHR CRC32: 0x0897021B
[iNes] Mapper: 69 Sub:0
[iNes] PRG ROM: 128 KB
[iNes] CHR ROM: 256 KB
```

I've tried toggling a number of settings and couldn't reproduce this, either.
I originally did this by running to the door on the first stage, and I thought using the save state you provided would cause it, but I also get the proper screen.

So, I'm actually not sure what's going on here - I'm assuming you have some option enabled somewhere that is causing this to break, though.

**Comment by Zet-sensei** [comment]
Thanks for the fast response you two.

I've redumped my card just minutes ago & removed manually the iNES header just to be sure (CRC32 - 0x0897021B, as expected per Bootgood DB).
The problem remained.

I've redone completely my Mesen config from scratch.
The problem was an overclocking option that was left from previous testing sessions (After NMI >> 0) and not being removed by mistake for the current session.

I've concluded too fast on a bug inside the emulator, as the game was fine on other emus and forgotten to verify my configuration.

The game is very sensitive to any overclocking setting.

Everything is fine now on the game on my side, I'm closing the issue.
